### PR TITLE
Adding Name to the data (not in anything yet)

### PR DESCRIPTION
**Pull request by Athight**
adding name column:
ALTER TABLE offers ADD name VARCHAR(255);

### CODE REVIEW

**Comment by Athight** [comment]
cleaned up merge conflicts, shouldn't have changed anything particularly important

File: src/templates/start.html [hunk 1]
```diff
@@ -82,228 +84,330 @@
...
-	let autoMin = true;
-	let captchaValid = false;
-	let offerValid = false;
-
-	function updateButton(errorMessage) {
-		if (captchaValid && offerValid) {
-			ui.btnSend.removeAttribute('disabled');
-		} else {
-			ui.btnSend.setAttribute('disabled', 'disabled');
+	<<<<<<< HEAD {%FILE=footer.snippet.html%} <script>
```
[the remark this answers is not in the document]
Comment (Athight) [reply src/templates/start.html]: 🙀 !#$%#$^ RIP my trust in VS code

**Comment by Athight** [comment]
seems i have different automatic whitespace resolution..
i can try to remove the change if you like?

File: src/entities.py [hunk 2]
```diff
@@ -320,9 +320,10 @@ def __init__(self, row):
 		self.created_ts = row['created_ts']
 		self.expires_ts = row['expires_ts']
 		self.confirmed = row['confirmed']
+		self.name = row['name']
```
Comment (bigfighter) [inline src/entities.py]: it's already there in line 314.

**Pull request merged**